tests/sitecustomize.py

```diff
@@ -1,5 +1,6 @@
-import coverage
+import os
 
-print("\033[31menabling coverage\033[0m")
+print(f"\033[31menabling coverage: {os.environ.get('COVERAGE_PROCESS_START', 'unset')}\033[0m")
```

Comment:
Interesting how this library works by relying on setting the 'COVERAGE_PROCESS_START' env var. Needless to say that we should make sure that this happens for our tests